### ISSUE TITLE
CoeCLI Installation clarifications

### ISSUE THREAD

**Issue by sefa-phaag** [MicrosoftDocs/power-platform#1807]
For step 1 on this page under "To install the CoE CLI" it would probably be good to clarify that this means the "Source Code.zip" file as the GitHub landing page referenced has several zip files, including one specifically called "coe-cli.zip" that does not work with the rest of the instructions.

Furthermore, is the purpose of the coe-cli.zip documented anywhere? It contains a coe-cli.exe, which I assume is a prebuilt version of the CoE CLI, but I couldn't find any documentation on how to use it. All of the code examples use just "coe", so it's not being called directly, and running the exe itself does nothing.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 6afd0cd6-006d-2efc-2702-a7daf5b92d67
* Version Independent ID: 6afd0cd6-006d-2efc-2702-a7daf5b92d67
* Content: [Center of Excellence (CoE) Command Line Interface (CLI) installation - Power Platform](https://learn.microsoft.com/en-us/power-platform/guidance/coe/cli/install)
* Content Source: [power-platform/guidance/coe/cli/install.md](https://github.com/MicrosoftDocs/power-platform/blob/main/power-platform/guidance/coe/cli/install.md)
* Service: **power-platform**
* Sub-service: **guidance**
* GitHub Login: @Grant-Archibald-MS
* Microsoft Alias: **grarchib**

**Comment by bishalgoswami** [comment]
Thanks for your comment, @sefa-phaag. This issue has been assigned to the doc author for review.

**Comment by sciencestudent-Sharon** [comment]
Yes, please update this for clarification, because I also ran into a similar issue of just downloading the coe-cli.zip file rather than the source file.
